sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -172,25 +175,37 @@ private static File findTsConfig(InputFile inputFile, File projectBaseDir) {
     ProcessBuilder processBuilder = new ProcessBuilder(command.commandLineTokens());
     setNodePath(localTypescript, processBuilder);
     LOG.debug(String.format("Starting external process `%s`", commandLine));
+    Process process = null;
     try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2093)